scripts/cnn_variant_wdl/cram2filtered.wdl

```diff
@@ -0,0 +1,473 @@
+# This workflow takes an input CRAM to call variants with HaplotypeCaller and filter the calls with a neural net
+workflow Cram2FilteredVcf {
+    File input_cram                 # Aligned CRAM file
+    File reference_fasta 
+    File reference_dict
+    File reference_fasta_index
+    File architecture_json          # Neural Net configuration for CNNScoreVariants  
+    File architecture_hd5           # Pre-Trained weights and architecture for CNNScoreVariants
+    Int inference_batch_size        # Batch size for python in CNNScoreVariants  
+    Int transfer_batch_size         # Batch size for python in CNNScoreVariants
+    String output_prefix            # Identifying string for this run will be used to name all output files
+    String tensor_type              # What kind of tensors the Neural Net expects (e.g. reference, read_tensor)
+    File? gatk_override
+    String gatk_docker    
+    File picard_jar  
+    File calling_intervals
+    Int scatter_count               # Number of shards for parallelization of HaplotypeCaller and CNNScoreVariants
+    String extra_args               # Extra arguments for HaplotypeCaller
+
+    # Runtime parameters
+    Int? mem_gb
+    Int? preemptible_attempts
+    Int? disk_space_gb
+    Int? cpu 
+
+    call CramToBam {
```

Comment:
Can you put a conditional on this in case there's bam input?  I don't know if there's a spiffy WDL way to query the file extension.

Reply:
It's a bit ugly, but I'm doing this via `select_first` now